article.js

```diff
@@ -0,0 +1,36 @@
+var collectedEntries = [];
+var blog = {};
+blog.rawData = [];
+var $template = $('#template').clone();
+
+var Article = function(blog) {
+  this.title = blog.title;
+  this.category = blog.category;
+  this.author = blog.author;
+  this.authorUrl = blog.authorUrl;
+  this.publishedOn = blog.publishedOn;
+  this.body = blog.body;
+  collectedEntries.push(this);
+}
+
+Article.prototype.toHTML = function() {
+  $('.title').text(this.title);
+  $('.author').text(this.author);
+  $('.authorUrl').text(this.authorUrl);
+  $('.category').text(this.category);
+  $('.publishedOn').text(this.publishedOn);
+  $('.body').text(this.body);
+  $
+  // return '<article>' +
+  // '<h2>' + this.title + '</h2>' +
+  // '<h5>' + this.publishedOn + '</h5>' +
+  // '<li>' + this.author + ' ' + this.authorUrl+ '</li>' +
+  // this.body +
+  // '</article>'
```

Comment:
Try and avoid commented out code like this when pulling into master. You want to try and keep your master code as clean as possible. If you need to go back and see what you had in the past, you can always use git to rewind to any previous point. That's one of its greatest strengths.